### PR TITLE
Added Login page and 2nd page after login

### PR DESCRIPTION
**Pull request by ghost**
Code Review for Log-In Page
Other parts is on Progress

### CODE REVIEW

**Comment by ghost** [comment]
@junrezapico , You may wonder that there are props that I have changed to string, it is because the console is throwing warning that the DOM should not have that kind of data and suggested that I should use string or number. Also they don't accept camel cases

customProps

File: src/components/atoms/button/style/button.styled.component.tsx [hunk 1]
```diff
@@ -7,13 +7,13 @@ export const Styledbutton = styled(Button)<buttonInterface>`
   ${fontLink.url};
   ${fontLink.fontFamily};
   border-radius: 4px;
-  ${props => props.isPrimaryProp? '' : `
+  ${props => props.isprimaryprop === 'true'? '' : `
```
Comment (junrezapico) [inline src/components/atoms/button/style/button.styled.component.tsx]: i'm not entirely sure but I think the reason why this is throwing the DOM warning you mentioned is because you have to type the props object here. Have you tried doing so?

File: src/components/pages/logInPage/index.tsx [hunk 2]
```diff
@@ -0,0 +1 @@
+export { default } from './logInPage';
```
Comment (junrezapico) [inline src/components/pages/logInPage/index.tsx]: I think it's fair to use just 'login' as the path here instead of 'logInPage' since it's already in the '/pages' directory. what do you think?

File: src/stories/templates/searchHead/searchHeadTemplate.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+import SearchHead, {IsearchHeadForm} from '../../../components/templates/searchForm';
+
+
+export default {
+    title: 'Components/Templates/SearchHead',
+    component: SearchHead,
+
+  } as Meta;
+
+  const Template: Story<IsearchHeadForm> = (args:IsearchHeadForm) => <SearchHead {...args} />;
```
Comment (junrezapico) [inline src/stories/templates/searchHead/searchHeadTemplate.stories.tsx]: The spacing here looks weird to me. But this will be automatically solved when we will be using a linter so could you please adjust the spacing manually for now?

**Review by junrezapico — CHANGES_REQUESTED** [review]

File: src/components/atoms/button/style/button.styled.component.tsx (hunk 1 above)
Comment (ghost) [reply src/components/atoms/button/style/button.styled.component.tsx]: Yes first warning is that they don't accept camelCase props as we can see on this commit, second warning is they suggest that I should pass string or number instead of Boolean values, though it is a warning only the app still compiled and running, though I checked the docs on react it is suggested as a best practice. If we ignore this it can lead a potential security risks to the app once deployed on prod.

Comment (ghost) [reply src/components/atoms/button/style/button.styled.component.tsx]: To be more elaborate here's the 2nd warning after I fix the camelCase warning..

Console Warning / error
console

Though it is said that it not a boolean attribute, i set it as boolean in the interface.
bool

so thats why I applied the reacts suggestion to clear out the warnings and to apply whats is best practice in accordance with the docs

Comment (junrezapico) [reply src/components/atoms/button/style/button.styled.component.tsx]: @ClydeIanVananaz What happens if you specify the "props" variable here? Something like ${(props: {isPrimary?: boolean;}) => props.isPrimary ?  ...

Comment (ghost) [reply src/components/atoms/button/style/button.styled.component.tsx]: Hmm I will try, I will get back to you for the feedback :)

Comment (ghost) [reply src/components/atoms/button/style/button.styled.component.tsx]: @junrezapico 

I have tried you suggestion but react doesnt accept that line of code.


I Tried also to be more specific by using the interface,


But it has same warning / error showed in console.

File: src/App.tsx [hunk 4]
```diff
@@ -1,25 +1,25 @@
...
+          <Route path="/selecttodo" component={SelecTodoPage} />
+        </Switch>
+      </Router>
     </div>
   );
 }
 
+const LoginPage = () => <LogInPage />;
+const TaskPage = () => <TasksPage />;
+const SelecTodoPage = () => <SelectToDoComponent />;
```
Comment (junrezapico) [inline src/App.tsx]: I think its better to declare these at the top so the file can be read from top to bottom

File: src/components/atoms/button/button.tsx [hunk 5]
```diff
@@ -1,38 +1,40 @@
...
-      color= {color}
+      color={color}
       primary={isPrimary}
       basic={!isPrimary}
       loading={isLoading}
       disabled={isDisabled}
-      isPrimaryProp={isPrimary}
-      isDisabledProp={isDisabled}
+      isprimaryprop={isPrimary ? isPrimary : undefined}
+      isdisableddrop={isDisabled ? isDisabled : undefined}
```
Comment (junrezapico) [inline src/components/atoms/button/button.tsx]: If this concern will need more time to fix then I think it's better to put it in a separate pull request. this is so each of your pull requests wont be as large as this one (40+ files modified). Do you think we should just separate this issue?

**Review by junrezapico — APPROVED** [review]
